### PR TITLE
Monitoring Policies Create Panel - Create New Policy

### PR DESCRIPTION
**Pull request by faizanjanjua**
## Description
When use clicks “Create Policy” on the Edit Panel, navigate to monitoring-policies-new/create and the MonitoringPolicyCreatePanel Vue component. Can reuse much of the design of the MonitoringPolicyEditPanel.

Implement the following:

Call monitoringPolicyStore.copyPolicy(defaultPolicy) to create a new non-default, empty policy and set it as the selected policy

Route to /monitoring-policies-new/create, this will load the MonitoringPolicyCreatePanel component

Basic Information and Alert Rules tabs. Alert Rules will be blank for now

Name & Description, Basic Info component

Notifications

Tags - can be readonly for now

Cancel and Next buttons

Cancel clears the selected policy and navigates back to main table - /monitoring-policies/new

Next navigates to Alert Rules panel
## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2427

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

File: ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+</template>
+
+<script setup lang="ts">
+import { useMonitoringPoliciesStore } from '@/store/Views/monitoringPoliciesStore'
+import { Policy } from '@/types/policies'
+import router from '@/router'
+
+const store = useMonitoringPoliciesStore()
+const route = useRoute()
+let activeTab = ref(0)
```
Comment (synqotik) [inline ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue:28]: Should be `const`. Also, change name to `selectedTab`, seems that is what we use elsewhere.

File: ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue [hunk 2]
```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="monitoring-policies-edit-panel">
+    <div class="page-headline1">Monitoring Policies</div>
+    <FeatherTabContainer class="mt-2 " :model-value="1">
```
Comment (synqotik) [inline ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue:4]: Change `model-value` to `selectedTab`

File: ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+const route = useRoute()
+let activeTab = ref(0)
+const handleCancel = () => {
+  router.push('/monitoring-policies-new/')
+}
+const deleteMsg = computed(() =>
+  `Deleting monitoring policy ${store.selectedPolicy?.name} removes ${store.numOfAlertsForPolicy} associated alerts. Do you wish to proceed?`
+)
+const handleNext = () => {
+  console.log('activeTab.value++')
```
Comment (synqotik) [inline ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue:36]: Remove this `console.log` and replace with `selectedTab.value = 1`. Can use `0` and `1` for tab values.

File: ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue [hunk 4]
```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="monitoring-policies-edit-panel">
+    <div class="page-headline1">Monitoring Policies</div>
+    <FeatherTabContainer class="mt-2 " :model-value="1">
+      <template v-slot:tabs>
+        <FeatherTab >Basic Information</FeatherTab>
```
Comment (synqotik) [inline ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue:6]: Remove space in `<FeatherTab >`

File: ui/src/containers/MonitoringPolicies.vue [hunk 5]
```diff
@@ -34,7 +34,7 @@ const displayDetails = ref(false)
 const store = useMonitoringPoliciesStore()
 
 const onCreatePolicy = () => {
-  console.log('onCreatePolicy clicked')
+  store.copyPolicy({id: '', isDefault: true, tags: [], name: '', memo: '' })
```
Comment (synqotik) [inline ui/src/containers/MonitoringPolicies.vue:37]: `isDefault` should be `false`. There's one default policy that is "built in", everything else should have `isDefault` as `false`.

**Review by synqotik — CHANGES_REQUESTED** [review]
Requesting a few changes.

File: ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue (hunk 1 above)
Comment (faizanjanjua) [reply ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue:28]: done

File: ui/src/containers/MonitoringPolicies.vue (hunk 5 above)
Comment (faizanjanjua) [reply ui/src/containers/MonitoringPolicies.vue:37]: done

File: ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue (hunk 4 above)
Comment (faizanjanjua) [reply ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue:6]: done

File: ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue (hunk 2 above)
Comment (faizanjanjua) [reply ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue]: i have changed this to :modelValue="selectedTab" 
so now all our conditions are working .. and this jira all tasks are now completed as per working of this task

File: ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue (hunk 3 above)
Comment (faizanjanjua) [reply ui/src/components/MonitoringPolicies/MonitoringPolicyCreatePanel.vue]: done

**Review by synqotik — APPROVED** [review]
Thanks for changes.

**Pull request merged**